Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -950,433 +906,440 @@
-    <RegEx find="(stomak)(u|om)?" replaceWith="trbuh$2" />
-    <RegEx find="(stomač)(na|ne|ni|no|nu|noj|nom|nima)" replaceWith="trbuš$2" />
-    <RegEx find="\b([sS])(trj?el)(a|e|i|u|o|om|ac|cu|ci|cima|ama)" replaceWith="$1trijel$3" />
-    <RegEx find="(sujeverj)(a|e|u|ima|em)" replaceWith="praznovjerj$2" />
-    <RegEx find="(s|S)used(a|e|i|o|u|ama|ima|ski|skim|ima)?" replaceWith="$1usjed$2" />
-    <RegEx find="(suštin)(e|i|om)" replaceWith="biti" />
-    <RegEx find="(s|S)vat(am|aš|a|amo|ate|aju|ajući|ali|ale|alo|aše)" replaceWith="$1hvać$2" />
-    <RegEx find="(s|S)vedo(k|ke|ka|ku|kom|ci|cima|kinja|kinju|kinji|čanstvo|čanstva|čili|čio|čila|čiti)" replaceWith="$1vjedo$2" />
-    <RegEx find="(s|S)vesn(a|e|i|o|u|om|oj|ima|ost|ošću)" replaceWith="$1vjesn$2" />
+    <RegEx find="([pP])esnik(a|u|ov|ovu|om)?\b" replaceWith="$1jesnik$2" />
+    <RegEx find="\b([pP])obed(a|i|e|u|o|om|ama)" replaceWith="$1objed$2" />
+    <RegEx find="\b([pP])obed(im|iš|i|imo|ite|e|iti|ili|ivši)" replaceWith="$1obijed$2" />
+    <RegEx find="([pP])obeg(ao|la|li|le|lo|avši)" replaceWith="$1objeg$2" />
+    <RegEx find="([pP])otcj?enjen(a|e|i|o|u|om|oj|om|ima)?\b" replaceWith="$1odcijenjen$2" />
+    <RegEx find="([pP])oent(a|e|u|i|o|om|ama)" replaceWith="$1oant$2" />
+    <RegEx find="([pP])ogrj?eš(io|ila|ili)" replaceWith="$1ogriješ$2" />
+    <RegEx find="([pP])oslednj(i|im|e|em|eg|ima)" replaceWith="$1osljed$2" />
+    <RegEx find="pomera(m|š|mo|te)" replaceWith="miče$1" />
+    <RegEx find="([pP])overenj(e|a|u|em)" replaceWith="$1ovjerenj$2" />
+    <RegEx find="([pP])overljiv(a|e|i|o|u|ima|ima|om|og|oj)?\b" replaceWith="$1ovjerljiv$2" />
+    <RegEx find="\b([pP])revoz(a|u|om|ovi|ovima)?\b" replaceWith="$1rijevoz$2" />
+    <RegEx find="([pP])retnj(a|e|i|o|u|om|ama)?\b" replaceWith="$1rijetnj$2" />
+    <RegEx find="([pP])rimi?j?ećuj(em|eš|e|emo|ete|u)" replaceWith="$1rimjećuj$2" />
+    <RegEx find="([pP])rimer(ak|en|ena|eno|enog|enu|enim|ku|kom|ci|cima)?\b" replaceWith="$1rimjer$2" />
+    <RegEx find="([nN])eprimer(en|ena|eno|enog|enu|enim)" replaceWith="$1rimjer$2" />
+    <RegEx find="\b([pP])rimet(an|na|no|ljiv|ljiva|ljivo|ljivost|ljivošću|ljivosti|nost)" replaceWith="$1rimjet$2" />
+    <RegEx find="\b([pP])rimen(a|e|i|o|u|om|jena|jene|jenoj|jenim|jiv|jiva|jivo|jivog|jivom|jivu)?\b" replaceWith="$1rimjen$2" />
+    <RegEx find="\b([pP])rimeni(m|š|imo|ite|ti|li|la|le|lo)" replaceWith="$1rimijeni$2" />
+    <RegEx find="\b([pP])rimeti(o|la|le|lo|li)?\b" replaceWith="$1rimijeti$2" />
+    <RegEx find="([pP])rosečn(a|e|i|o|u|om|oj|im|ima)?\b" replaceWith="$1rosječn$2" />
+    <RegEx find="poen(i|e|ima)" replaceWith="bodov$1" />
+    <RegEx find="([pP])over(im|iš|i|imo|ite|e|ili|ile|ilo|iše)" replaceWith="$1ovjer$2" />
+    <RegEx find="([pP])rover(a|e|i|o|u|om|im|ama|im|iš|i|imo|ite|iti|e|ili|ile|ilo|iše)" replaceWith="$1rovjer$2" />
+    <RegEx find="([pP])rjedlo(g|ga|ge|gu|gom|zi|zima)" replaceWith="$1rijedlo$2" />
```

Comment:
Probably not:
```
PS> [regex]::Replace('predlozima', '([pP])rj?edlo[gz](a|e|u|om|i|ima)', '$1rijedlo$2')
prijedloima
```
The "z" has disappeared.
I'll use
`<RegEx find="([pP])rj?edlo([gz])(a|e|u|om|i|ima)" replaceWith="$1rijedlo$2$3" />`
Which keeps the "g" or the "z".